zds/tutorialv2/views/published.py

```diff
@@ -1122,6 +1122,10 @@ def get_urls_to_render(self):
     def get_content_urls(self):
         content = self.versioned_object
         urls = [NamedUrl(content.title, content.get_absolute_url_online(), 0)]
+        nb_of_pages = self.object.get_note_count() / settings.ZDS_APP['content']['notes_per_page']
```

Comment:
Pour corriger l'erreur remontée par firm1 :

```suggestion
        nb_of_pages = ceil(self.object.get_note_count() / settings.ZDS_APP['content']['notes_per_page'])
```

en ajoutant `from math import ceil` au début